WeakAurasOptions/OptionsFrames/MoverSizer.lua

```diff
@@ -101,6 +101,34 @@ local function ConstructMover(frame)
   return top, bottom, left, right
 end
 
+local function PositionMover(frame)
+  if frame.moveTop:GetLeft() then
```

Comment:
I’m not asking you to change this, but can you clarify what moveTop and moveBottom refer to? Are those the buttons which nudge the aura up and down?